gui/src/components/HomeComponents/CreateOrEditProjectBody.tsx

```diff
@@ -261,6 +358,77 @@ export default function CreateOrEditProjectBody({
               />
             )}
           </div>
+
+          <div>
+            <label className="secondary_color mb-1 block text-sm font-medium">
+              Repository
+            </label>
+            <div className="flex items-center space-x-4">
+              <label className="flex items-center">
```

Comment:
this radio component is way to length, make it into a component and use it here. create a dir for it and module css for the same component

Reply:
Doesn't make sense, this radio button has a defined structure no evidence it'll be reusable later.